cmd/nvidia-container-runtime-hook/container_config_test.go

```diff
@@ -843,10 +852,17 @@ func TestGetDevicesFromEnvvar(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.description, func(t *testing.T) {
-			image, _ := image.New(
+			opts := []image.Option{
```

Comment:
The tests in `TestGetDevicesFromEnvvar` can be moved to `cuda_image_test.go` and renamed `TestGetVisibleDevices`. (note that strictly speaking this is only testing get visible devices from envvar unless we expand the test cases to also set the other options.